src/__tests__/components/WalletListCurrencyRow.test.js

```diff
@@ -0,0 +1,39 @@
+/* globals describe it expect */
+/* eslint-disable flowtype/require-valid-file-annotation */
+
+import * as React from 'react'
+import ShallowRenderer from 'react-test-renderer/shallow'
+
+import { getTheme } from '../../components/services/ThemeContext.js'
+import { WalletListRowComponent as Request } from '../../components/themed/WalletListCurrencyRow.js'
+
+describe('Request', () => {
+  it('should render with loading props', () => {
+    const renderer = new ShallowRenderer()
+
+    const props = {
+      currencyCode: 'string',
+      gradient: true,
+      onPress: () => undefined,
+      onLongPress: () => undefined,
+      showRate: true,
+      paddingRem: 11 | [11],
```

Comment:
This should not have the 'or' sign, it was only part of the type.
You should just pass one of the options.